source/administration/changelog.md

```diff
@@ -2,9 +2,14 @@
 
 This changelog summarizes updates to [Mattermost Team Edition](http://www.mattermost.org/), an open source team messaging solution released bi-monthly under an MIT license, and [Mattermost Enterprise Edition](https://about.mattermost.com/pricing/), a commercial upgrade offering enterprise messaging for large organizations.
 
-## Release v3.6.0  
+## Release v3.6.1  
 
-Release date: 2017-01-16
+### Notes on Patch Release
+
+ - **v3.6.1, released 2017-01-17**
+   - Fixed a performance regression when sending many notifications at once, upgrading only necessary if you notice a large delay when sending `@all` or `@channel` notifications in channels with a lot of users
```

Comment:
I thought the second part, `upgrading only necessary...`, was still referring to the fix itself.

Maybe we could break it into two bullet points so it's more obvious 3.6.1 upgrade is only necessary for the case you described:

```
 - **v3.6.1, released 2017-01-17**
   - Fixed a performance regression when sending many notifications at once
   - Note: Upgrading to Mattermost v3.6.1 only necessary if you notice a large delay when sending `@all` or `@channel` notifications in channels with a lot of users
```